rfc000-rfc-process.md

```diff
@@ -116,6 +116,18 @@ the original one should be marked as Replaced.
 * **Final** – The RFC has been implemented. *(Standards Track type only)*
 * **Replaced** – The RFC has been superseded by another RFC.
 
+### Changing an Accepted RFC
+
+An accepted RFC may be modified in two ways, depending on the type of RFC:
+
+1) To support software implementations meeting the specifications of an RFC,
+Standards Track RFCs must be replaced by a new RFC. This is done by specifying
```

Comment:
Standards Track can be edited as needed. Only Final RFCs are locked down because that means the feature or change is in a release build.